### DISCUSSION TITLE
ESP32 M5Stack ATOM LITE & SGP30 - ABS Humidity - crypt. Value

### DISCUSSION THREAD

**Discussion by elbman** [arendst/Tasmota#13621] [Q&A] [answered]
I'm using M5Stack Atom Lite with SGP30 Sensor (latest  Tasmota Release 10)

I2C Sensor was regognized automatically and shows me needed TVOC & eCO2

I'm a bit surprised that there is now listed a third value for ABS Humidiy, showing somethin like ?��?��?��?( �? g/m3

![M5](https://user-images.githubusercontent.com/36014482/140927632-a6b823f1-d796-457d-892a-9304771587c8.jpg)

what's going wrong here?

**Comment by barbudor** [comment, marked as answer]
Fixed with https://github.com/arendst/Tasmota/pull/13624
Will be available in development after merge

**Reply by elbman** [reply]
👌
